README.md

```diff
@@ -5,7 +5,31 @@ discosrv
 
 This is the global discovery server for the `syncthing` project.
 
-`go get github.com/syncthing/discosrv`
+To get it, run `go get github.com/syncthing/discosrv` or download the
+[latest build](http://build.syncthing.net/job/discosrv/lastSuccessfulBuild/artifact/)
+from the build server.
 
-Or download the latest [Linux build](http://build.syncthing.net/job/discosrv/lastSuccessfulBuild/artifact/).
+Usage
+-----
 
+The discovery server supports `ql` and `postgres` backends.
+Specify the backend via `-db-backend` and the database DSN via `-db-dsn`.
+
+By default it will use in-memory `ql` backend. If you wish to persist the
+information on disk between restarts in `ql`, specify a file DSN:
+
+```bash
+$ discosrv -db-dsn="file://var/run/discosrv.db"
+```
+
+For `postgres`, you will need to create a database and a user with permissions
+to create tables in it, then start the discosrv as follows:
+
+```bash
+$ discosrv -db-backend="postgres" -db-dsn="postgres://user:password@localhost/databasename"
```

Comment:
For psql at least, having the DSN in an environment variable has the advantage that the password isn't visible in `ps` and so on to other users. For my use case this isn't a big deal as there's noone else on the discovery server, but as a best practice thing I think it may still be preferable. Perhaps keep the `-db-dsn` but let it default to `DISCOSRV_DB_DSN` or something if not set on the command line?